Src/main.c

```diff
@@ -315,34 +315,15 @@ uint16_t target_e_com_time_low;
 uint8_t compute_dshot_flag = 0;
 uint8_t crsf_input_channel = 1;
 uint8_t crsf_output_PWM_channel = 2;
-char eeprom_layout_version = 2;
-char dir_reversed = 0;
-char comp_pwm = 1;
-char VARIABLE_PWM = 1;
-char bi_direction = 0;
-char stuck_rotor_protection = 1; // Turn off for Crawlers
-char brake_on_stop = 0;
-char stall_protection = 0;
-char use_sin_start = 0;
-char TLM_ON_INTERVAL = 0;
+char eeprom_layout_version = 3;
```

Comment:
I think this PR should not actually change the layout, so no change in version, so we can test just the structure change independent of feature change

Reply:
reverted